CMakeLists.txt

```diff
@@ -1145,18 +1142,23 @@ add_library (corepp
     src/ripple/crypto/impl/ec_key.cpp
     src/ripple/crypto/impl/openssl.cpp
   >)
-add_library (Ripple::corepp ALIAS corepp)
-target_compile_options (corepp
+add_library (Ripple::xrpl_core ALIAS xrpl_core)
+target_include_directories (xrpl_core
+  PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/ripple>
+    $<INSTALL_INTERFACE:include>)
+target_compile_options (xrpl_core
   PUBLIC
     $<$<BOOL:${is_gcc}>:-Wno-maybe-uninitialized>)
```

Comment:
I know you didn't touch this here, but just commenting that this seems a bit odd that this flag is applied only to GCC.